frontend/src/static/js/v_zoom.js

```diff
@@ -1,15 +1,42 @@
 window.vZoom = {
   props: ['info'],
   template: window.$('v_zoom').innerHTML,
-  data: () => ({}),
+  data() {
+    return {
+      filterTimeFrame: window.hashParams.timeframe,
+    };
+  },
   methods: {
     openSummary() {
       this.$emit('view-summary', {});
     },
 
+    filterCommits() {
+      const { user } = this.info;
+      const filtered = user.commits.filter(
+          (commit) => commit.date >= this.info.sinceDate && commit.date <= this.info.untilDate,
+      );
+      user.commits = filtered;
+    },
+
     getSliceLink(slice) {
       return `${window.getBaseLink(this.info.user.repoId)}/commit/${slice.hash}`;
     },
+
+    updateUser(repos) {
```

Comment:
Do we need to pass the `repos` in as an argument? Can't it be directly read?

Reply:
U mean directly read from `this.filtered` in `v_summary`? I don't think so as they are separate components that have no parent/child relation.